NuGet.config

```diff
@@ -2,8 +2,6 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
-    <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
-    <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <!-- Restore sources should be defined in build/sources.props. -->
```

Comment:
Does `RestoreSources` clear \ override sources specified in global NuGet.config? If so, we shouldn't need this file anymore.

Reply:
Not sure. regardless, I want to leave this feed here to ensure we don't pickup feeds from other places in tests or sample projects that my not import DB.props.